copyright

```diff
@@ -2114,6 +2114,44 @@ Files:
 Copyright: Zitchas (zitchas.jma@gmail.com)
 License: CC-BY-SA-4.0
 
+Files:
+ images/effect/incipias flare/flare*
+ images/effect/plasma particle.png
+ images/effect/tractor impact*
+ images/projectile/plasma discharge+.png
+ images/projectile/strange ray/*
+ images/projectile/tractor beam*
+ images/outfit/gas class shield.png
+ images/outfit/liquid class shield.png
+ images/outfit/liquid hydrogen cell.png
+ images/outfit/tractor beam.png
+ images/hardpoint/tractor beam.png
+ images/planet/pontes.png
+Copyright: RisingLeaf (https://github.com/RisingLeaf)
```

Comment:
Usually you put only your name or name and email. (I think)

Reply:
I don't want to share my email and this is what I do instead, this is fine as is

Reply:
OK.